Automattic/jetpack
# Option: Update `featured_image_email_enabled` option name to `wpcom_featured_image_in_email`

#### Changes proposed in this Pull Request:

* rename `featured_image_email_enabled` option to `wpcom_featured_image_in_email`

The new name:
- better represents the option usage
- includes `wpcom_` prefix to communicate it is WPCOM-related and to prevent potential name conflicts with third-party plugins

The option hasn't been used in production yet, so there are no consequences of this change to our users.

Related issue / task: https://github.com/Automattic/wp-calypso/issues/71070

#### Other information:


#### Jetpack product discussion


- Featured Image Template Toggle (pdDOJh-Nu-p2)
- Modernize Reading Settings (pdDOJh-17o-p2)

#### Does this pull request change what data or activity we track or use?


No, it does not.

#### Testing instructions:

1. For testing on WPCOM, sandbox `public-api.wordpress.com` and apply the PR to your sandbox by running the [script below](https://github.com/Automattic/jetpack/pull/27955#issuecomment-1352848086); For testing on a Jetpack site, you can use Jurassic Ninja and then sync built Jetpack using `rsync` (PCYsg-eg0-p2); For testing on Atomic, these steps can be followed: pb5gDS-1rQ-p2.
3. Use WPCOM developer console and request `/sites/<site_url_or_id>/settings` endpoint.
4. The new option name (`wpcom_featured_image_in_email`) should be included in the API response:
![Markup on 2022-12-15 at 11:42:27](https://user-images.githubusercontent.com/25105483/207839006-a4897811-fe5e-4e3c-8427-8d6c7d12048a.png)
